Tribler/community/market/community.py

```diff
@@ -1657,49 +1638,43 @@ def send_transaction_completed(self, transaction, block):
         match_payload = self.incoming_match_messages[transaction.match_id]
         del self.incoming_match_messages[transaction.match_id]
 
-        linked_block = self.market_database.get_linked(block) or block
-        self.send_block_pair(block, linked_block, address=self.lookup_ip(match_payload.matchmaker_trader_id))
+        linked_block = self.trustchain.persistence.get_linked(block) or block
 
-    def on_transaction_completed_message(self, block1, block2):
-        tx_dict = block1.transaction
+        global_time = self.claim_global_time()
+        dist = GlobalTimeDistributionPayload(global_time).to_pack_list()
+        payload = HalfBlockPairPayload.from_half_blocks(block, linked_block).to_pack_list()
+        packet = self._ez_pack(self._prefix, 22, [dist, payload], False)
```

Comment:
22 = Magic number ?

Reply:
Yeah, this is a convention that I followed from the Dispersy era. I will use some defined constants in a subsequent PR 👍